docs/instroom/sip/minimale_sip_0.1.md

```diff
@@ -127,14 +125,25 @@ root_folder
 
 ## Structure of a meemoo SIP: bag-level
 
-test
+A bag is a compressed directory that serves as a wrapper around a SIP submitted by a CP for ingest in the meemoo archive. It conforms to the BagIt 1.0 specification ([RFC 8493](https://www.rfc-editor.org/rfc/rfc8493.html)).
+
+A bag only has a practical purpose as a transfer container between a CP's archive and meemoo's ingest space. The bag will be unpacked during ingest and will be deleted after processing. As such it will not appear in the meemoo archive as a separate entity.
 
-Gebruik, files erin, conformance aan bagit spec, doel
+**_Requirements_**
```

Comment:
Voor leesbaarheid: sorteer van MUST naar MAY. En technisch gezien zijn MAY's geen requirements :) Specs?

Reply:
Hoe zou je dit dan noemen, als je er ook MAY of SHOULD bij wil vermelden ipv. requirements? :)